internal/pkg/crypto/io.go

```diff
@@ -189,13 +188,17 @@ func (w *Writer) writeBlock(b []byte) error {
 		return w.err
 	}
 
-	w.writer.Write(iv)
+	if _, err := w.writer.Write(iv); err != nil {
+		return errors.Wrap(err, "failed to write IV")
+	}
 
 	encodedBytes := w.gcm.Seal(nil, iv, b, nil)
 
 	l := make([]byte, 4)
 	binary.LittleEndian.PutUint32(l, uint32(len(encodedBytes)))
-	w.writer.Write(l)
```

Comment:
linter complained: Error return value of a.monitor.Cleanup is not checked (errcheck)